### ISSUE TITLE
Edit plan details UI bug

### ISSUE THREAD

**Issue by Andreasgdp** [Andreasgdp/saving-plan#5]
![image](https://github.com/user-attachments/assets/a30c0eeb-dfcb-495b-92fd-d4dd6dfc70e8)

Overflowing in items. Cancel and save plan should be only ones in the bottom. Duplicate should be "within the modal" just above the cancel and confirm
